tests/classifiersFrameworks/test_tensorflow.py

```diff
@@ -17,20 +17,24 @@
 # SOFTWARE.
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import pytest
 import logging
 
-import tensorflow as tf
 import numpy as np
+import pytest
+import tensorflow as tf
+from tests.classifiersFrameworks.utils import (
+    backend_test_class_gradient,
+    backend_test_fit_generator,
+    backend_test_input_shape,
+    backend_test_layers,
+    backend_test_loss_gradient,
+    backend_test_nb_classes,
+    backend_test_repr,
+)
+from tests.utils import ExpectedValue
```

Comment:
I think we should keep the imports from `tests` after the imports from `art` because they are not third-party and to keep it consistent across all tests.